### PR TITLE
Upgraded Rails 4.2.2 and other related gems.

### PR DESCRIPTION
**Pull request by dblock**
There're numerous security vulnerabilities with older Rails and such. Upgrading world.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -41,7 +40,6 @@ group :doc do
   gem 'sdoc', require: false
 end
 
-gem 'annotated-rails'
```
Comment (dblock) [inline Gemfile]: Couldn't find a compatible version, nuked for now.

**Comment by ilyakava** [comment]
Would be good with merging if travis was okay. I thought this was automagical, but from the travis error, does it want a line in `travis.yml` that does `bundle exec rake db:create && bundle exec rake db:migrate`?

**Comment by dblock** [comment]
Now :green_heart:

**Comment by ilyakava** [comment]
Very nice! thank you! :boom:

**Pull request merged**